docroot/modules/custom/va_gov_facilities/src/EventSubscriber/FacilitiesSubscriber.php

```diff
@@ -33,6 +83,17 @@ public function entityPresave(EntityPresaveEvent $event): void {
     $this->clearNormalStatusDetails($entity);
   }
 
+  /**
+   * Entity update Event call.
+   *
+   * @param \Drupal\core_event_dispatcher\Event\Entity\EntityUpdateEvent $event
+   *   The event.
+   */
+  public function entityUpdate(EntityUpdateEvent $event): void {
+    $entity = $event->getEntity();
+    $this->archiveRelatedFacilityContent($entity);
```

Comment:
Since this method is specific to health care facilities, it would be good to make this more transparent.
```suggestion
    $this->archiveRelatedHealthFacilityContent($entity);
```